Tests/backend-specific-tests/ABS/DataLoaderPersistentABSWorkspaceTest.php

```diff
@@ -9,45 +9,18 @@
 use Keboola\DockerBundle\Docker\OutputFilter\OutputFilter;
 use Keboola\DockerBundle\Docker\Runner\DataLoader\DataLoader;
 use Keboola\DockerBundle\Docker\Runner\DataLoader\WorkspaceProviderFactory;
-use Keboola\DockerBundle\Docker\Runner\WorkingDirectory;
 use Keboola\DockerBundle\Exception\ApplicationException;
 use Keboola\DockerBundle\Tests\BaseDataLoaderTest;
-use Keboola\DockerBundle\Tests\Runner\BackendAssertsTrait;
 use Keboola\StorageApi\BranchAwareClient;
-use Keboola\StorageApi\Client;
 use Keboola\StorageApi\Components;
-use Keboola\StorageApi\Metadata;
 use Keboola\StorageApi\Options\Components\Configuration;
 use Keboola\StorageApi\Options\Components\ListConfigurationWorkspacesOptions;
 use Keboola\StorageApi\Workspaces;
 use Keboola\StorageApiBranch\ClientWrapper;
-use Keboola\Temp\Temp;
-use Psr\Log\NullLogger;
 use Psr\Log\Test\TestLogger;
 
 class DataLoaderPersistentABSWorkspaceTest extends BaseDataLoaderTest
 {
-    use BackendAssertsTrait;
-
-    private Client $client;
-
-    public function setUp(): void
-    {
-        parent::setUp();
-
-        $this->client = new Client([
-            'url' => getenv('STORAGE_API_URL_SYNAPSE'),
-            'token' => getenv('STORAGE_API_TOKEN_SYNAPSE'),
-        ]);
```

Comment:
Synapse pro tenhle test není potřeba. Důležité je aby byl ABS file workspace.

Envy jsem fixoval tu https://github.com/keboola/docker-bundle/pull/779 takže tohle se může vyhodit a může se použit klient z `BaseDataLoaderTest`.